account_invoice_export/tests/common.py

```diff
@@ -1,9 +1,9 @@
 # Copyright 2023 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
-from odoo.tests.common import TransactionCase
+from odoo.addons.base.tests.common import BaseCommon
```

Comment:
unless you really need to setup company, user, currency, etc, avoid using `BaseCommon`. It slows down the setup.